--- old/utils/update_archives.sh	2014-07-05 11:23:05.938224014 -0700
+++ new/utils/update_archives.sh	2014-07-05 11:30:31.590236462 -0700
@@ -135,12 +135,12 @@
 
 # fixme this is actually not enough to ensure that there is no cruft in the
 # test tree, since git may ignore files
-STATUS=`git status --porcelain`
-if [ ${#STATUS} -gt 0 ]
-then
-    echo "working tree is dirty, run from a clean working tree"
-    exit 1
-fi
+# STATUS=`git status --porcelain`
+# if [ ${#STATUS} -gt 0 ]
+# then
+#     echo "working tree is dirty, run from a clean working tree"
+#     exit 1
+# fi
 
 # This is a simulation of git symbolic-ref --short -q HEAD, which doesn't work with
 # older versions of git.  The --short option is supported by git-1.9.1.
